glibc does not define SA_RESTART when _POSIX_SOURCE is defined, so we
have to just assume the default signal.h include defines SA_RESTART

http://sourceware.org/bugzilla/show_bug.cgi?id=12252

--- rp-pppoe-3.10/src/pppoe.h
+++ rp-pppoe-3.10/src/pppoe.h
@@ -19,10 +19,6 @@
 
 extern int IsSetID;
 
-#if defined(HAVE_NETPACKET_PACKET_H) || defined(HAVE_LINUX_IF_PACKET_H)
-#define _POSIX_SOURCE 1 /* For sigaction defines */
-#endif
-
 #include <stdio.h>		/* For FILE */
 #include <sys/types.h>		/* For pid_t */
 
